README.md

```diff
@@ -69,6 +69,7 @@ Usage:  <br>
 |`--nist` | validate against the NIST restrictions |
 |`--nist-ta3` | validate against the NIST hypothesis restrictions (implies `--nist`) |
 |`--abort[=num]` | Abort validation after `[num]` SHACL violations (num > 2), or three violations if `[num]` is omitted. |
+|`--depth[=num]` | Perform shallow validation in which each SHACL rule (shape) is only applied to `[num]` target nodes, or 50 nodes if `[num]` is omitted (requires -t). |
```

Comment:
omitting num will error during runtime and will result in files being skipped.

Reply:
I'm not getting an error when `[num]` is omitted, but it is using 1 instead of 50, as you say below.  Will fix in an upcoming commit.  It was poorly/erroneously copied from `--abort` where 3 is both the minimum and the default.